.taskcluster.yml

```diff
@@ -175,7 +175,8 @@ tasks:
                       - index.mozillavpn.v2.${project}.branch.${head_branch}.latest.taskgraph.decision-${cron.job_name}
                       - index.mozillavpn.v2.${project}.branch.${head_branch}.revision.${head_sha}.taskgraph.decision-${cron.job_name}
                       - index.mozillavpn.v2.${project}.branch.${head_branch}.revision.${head_sha}.taskgraph.cron.${ownTaskId}
-
+                else:
+                  - index.mozillavpn.v2.${project}.branch.${head_branch}.latest.taskgraph.decision
```

Comment:
I think this might cause Decision tasks from pull-requests to overwrite Decision tasks from pushes. I think we need to namespace these out to make sure they stay separate from the push ones, or it could cause issues.

Maybe something like `index.mozillavpn.v2.${project}.pr.${short_head_branch}.latest.taskgraph.decision`?

Also whatever we choose, we're going to need to update `ci-configuration` to grant the scope necessary to add this to the index.

Reply:
@ahal According to the previous error, `${short_head_branch}` isn't available.

Reply:
Ah, we need to define it then like this:
https://github.com/mozilla-mobile/firefox-android/blob/main/.taskcluster.yml#L142

If using `head_branch` *just works*, then maybe that's good enough, but this value comes from Github and it appears it can sometimes contain `refs/heads` (I'm not sure exactly under what circumstances though).